geode-redis/src/main/java/org/apache/geode/redis/internal/GeodeRedisService.java

```diff
@@ -77,7 +78,7 @@ private void startRedisServer(InternalCache cache) {
           String.format("Starting GeodeRedisServer on bind address %s on port %s",
```

Comment:
Since you're working on this, could you move the log message after the start so that we log the actual port used in the event that it is started on an ephemeral port. (Otherwise it just says `Starting... on port 0`).

Reply:
We currently have two log messages. One says "starting" and shows the requested configuration. The other says "started" and show the actual host and port. It seems like both are worth having. Does that resolve this conversation?